pkg/kudoctl/kudoinit/prereq/webhook.go

```diff
@@ -142,11 +149,66 @@ func (k KudoWebHook) resourcesWithSelfSignedCA() (*admissionv1beta1.MutatingWebh
 	return &iaw, &ws, nil
 }
 
-func validateCertManagerInstallation(client *kube.Client, result *verifier.Result) error {
-	if err := validateCrdVersion(client.ExtClient, "certificates.cert-manager.io", certManagerAPIVersion, result); err != nil {
+func (k *KudoWebHook) detectCertManagerVersion(client *kube.Client, result *verifier.Result) error {
+	extClient := client.ExtClient
+
+	// Cert-Manager 0.11.0+
+	if err := k.detectCertManagerCRD(extClient, certManagerNewGroup); err != nil {
+		return err
+	}
+	if k.certManagerGroup != "" {
+		return nil
+	}
```

Comment:
this is odd combo of funcs and state... this function needs intimate knowledge of the `detectCertManagerCRD` func... which is odd..  

there is a lot of refactor here IMO... 
1. `detectCertManagerVersion`  seems like a very specific func name when it could be a generic kube func
2. we should prefer the return of values rather than the change of values passed by ref.  It would be much easy to read and reason about IMO
3. it also seems strange that error handling is happening in 2 different ways... we have the return of an err and an adding of an error to the "result" which returns a nil.   It isn't clear when one vs the other is appropriate.